var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -368,6 +368,13 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
 
     # ###################### Dependencies ##########################
 
+    # External Kokkos
+    depends_on("kokkos@4.1.00", when="@14.4.0 +kokkos")
```

Comment:
```suggestion
    depends_on("kokkos@4.1.00", when="@14.4.0: +kokkos")
```
I think we should explicitly make this a range until the next `kokkos` update in `trilinos`